calculator/src/App.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class App extends Component {
+  state = { textArray: [], sentences: [], numberE: [], commas: [] };
```

Comment:
This is still really readable but you could consider breaking this object up line by line.

Reply:
I should never let 'prettier' have the final say on these things, you're very right. Thanks!